src/diff.jl

```diff
@@ -417,6 +417,23 @@ function jacobian_sparsity(du, u)
     sparse(I, J, true, length(du), length(u))
 end
 
+"""
+```julia
+jacobian_sparsity(op!,output::Array{T},input::Array{T}) where T<:Number
+```
+
+Return the sparsity pattern of the Jacobian of the mutating function `op!(output,input)`
+"""
+function Symbolics.jacobian_sparsity(op!,output::Array{T},input::Array{T}) where T<:Number
```

Comment:
splat the input to allow a variable number of them? IIRC SparsityDetection.jl does that.

Reply:
> and docs?
When I build the docs locally it shows up like this:

![Screenshot_20210910_095833](https://user-images.githubusercontent.com/9221937/132820855-18ec7cd6-6652-4433-b7a0-6410d9862917.png)

Do we need more or additional info at other places ?